mobile/src/main/res/values/strings.xml

```diff
@@ -220,6 +221,7 @@
     <string name="item_update_connection_error_label">%1$s update failed (No connection)</string>
     <string name="error_sending_alarm_clock_item_empty">Please enter a valid Item name</string>
     <string name="retry">Retry</string>
+    <string name="clear_errors">Clear errors</string>
```

Comment:
Maybe a better wording would be "Ignore" or "Cancel".

Reply:
Ignore sounds good.

Reply:
Right, "Cancel" sounds like it would stop sending further alarm clocks.